tests/e2e/machine_pool_test.go

```diff
@@ -425,6 +425,50 @@ var _ = Describe("TF Test", func() {
 				Expect(len(mpResponseBody.AvailabilityZones())).To(Equal(1))
 			})
 		})
+		Context("Author:amalykhi-High-OCP-65071 @OCP-65071 @amalykhi", func() {
```

Comment:
Per this test case we should create another subnet for the vpc and create the machinepool.

Reply:
@xueli181114 Do you mean create new subnet for the cluster VPC in the same AZ?

Reply:
In a different zone

Reply:
@xueli181114 will it be the negative test case? as this test case for single AZ cluster?

Reply:
this specific test case was created for the bug when subnet_id was blocked from using for singleAZ cluster